index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> Portfolio </title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <header class="mylogo">
+      <div class="left-logo"> 
+        <a href="#homepage">
+        <span class="logo">My Logo</span>
+        </a>
+      </div>
+      <nav class="right-logo">
+        <img src="/images/Bars/Normal Button/Tertiary/Icons/Union.png">
+      </nav>
+    </header>
+  <div class="img">
+    <section class="bio">
+      <header class="bio">
+        <h1 id="title">I’m Mariane Glad to see you!</h1>
```

Comment:
I actually used the same font size as figma, 40 px for the h1 and 16px for the p element, so I'm confuse, Maybe if you adjust the width of your browser it will look the same as it does in mine.

Reply:
@aimalamiri I actually used the same font size as figma, 40 px for the h1 and 16px for the p element, so I'm confuse, Maybe if you adjust the width of your browser it will look the same as it does in mine.